pose_format/pose_visualizer.py

```diff
@@ -27,6 +27,8 @@ def _draw_frame(self, frame: ma.MaskedArray, frame_confidence: np.ndarray, img)
       c = person_confidence.tolist()
       idx = 0
       for component in self.pose.header.components:
+        if component.name == 'POSE_WORLD_LANDMARKS':
```

Comment:
This specificity for the given component is not great design, in my opinion. 
The visualizer code does not really know anything about the pose, except for what is stored in the file. Having additional behavior for a specific case breaks principals in polymorphism.

I suggest that this would be removed, and you use, in your code:
```py
p.get_components(["FACE_LANDMARKS", "POSE_LANDMARKS", "LEFT_HAND_LANDMARKS", "RIGHT_HAND_LANDMARKS"])
```
before drawing.

I can also imagine a case where one would actually like to visualize these landmarks, and so will not understand why they don't show